display.c

```diff
@@ -0,0 +1,58 @@
+//
+// Created by Alex on 03.11.2019.
+//
+
+#include <stdio.h>
+#include <string.h>
+void displayfood(int nofood, char food[][100])
+{
+    printf("Please choose the food you feel like eating today:\n");
+    for (int i = 0; i < nofood; i++)
+    {
+        printf("%c) %s \n", 'a' + i, food[i]);
+    }
+    printf("%c) Go back \n>", 'a' + nofood);
+}
+void displayfoodtype(int nofood, char foodtype[][100], int price[], char food[])
+{
+    printf("Please choose the type of %s:\n", food);
+    for (int i = 0; i < nofood; i++)
+    {
+        printf("%c) %s: %d \n", 'a' + i, foodtype[i],price[i]);
+    }
+    printf("%c) Go back \n>", 'a' + nofood);
+}
+
+void displaydrink(int nodrink, char drink[][100],int price[], char food[])
+{
+    printf("Please choose a drink to go with your %s:\n" ,food);
+    for(int i=0;i<nodrink;i++)
+    {
+        printf("%c) %s: %d \n",'a'+i,drink[i],price[i]);
+    }
+    printf("%c) No, thanks!\n", 'a'+nodrink);
+    printf("%c) Go back \n>",'a'+nodrink+1);
+}
+void displaycutlery()
+{
+    printf("Do you want cutlery?\n");
+    printf("a) Yes\n");
+    printf("b) No thanks!\n");
+    printf("c) Go back!\n>");
+}
+void displayreceipt(char name[], char food[], int foodprice, char drink[], int drinkprice, int isdrink, int cutlery, char comment[])
```

Comment:
for me this naming convention in which you don't separate the words in any way is a bit hard to read. At least it's consistent, you don't have to change it if you prefer it like this